src/components/IconList/IconList.js

```diff
@@ -0,0 +1,125 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import List from '@geops/react-ui/components/List';
+import Button from '@geops/react-ui/components/Button';
+import { FaCaretUp, FaCaretDown } from 'react-icons/fa';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+
+import './IconList.scss';
+
+const propTypes = {
+  disabled: PropTypes.bool,
+  options: PropTypes.array.isRequired,
+  selected: PropTypes.string,
+  icons: PropTypes.object.isRequired,
+  onSelect: PropTypes.func.isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  disabled: false,
+  selected: null,
+};
+
+class IconList extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedOption: props.selected,
+      iconListVis: false,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    const { selected } = this.props;
+
+    if (selected !== prevProps.selected) {
+      this.select(selected);
+    }
+  }
+
+  select(option) {
+    this.setState({
+      selectedOption: option,
+    });
+  }
+
+  renderOption(option) {
+    const { icons, t, options } = this.props;
+
+    const opt = option || (options.length ? options[0] : null);
+
+    const label = (
+      <div className="wkp-zweitausbildung-routes-dropdown-label">{t(opt)}</div>
```

Comment:
I will change this (and the following lines) as zweitausbildung should not be part of `IconList`